src/libraries/System.Private.CoreLib/src/System/String.cs

```diff
@@ -236,7 +236,7 @@ private static unsafe string CreateStringForSByteConstructor(byte* pb, int numBy
                 throw new ArgumentException(SR.Arg_InvalidANSIString);
             return newString;
 #else
-            return Encoding.UTF8.GetString(pb, numBytes);
+            return CreateStringFromEncoding(pb, numBytes, Encoding.UTF8);
```

Comment:
Why is the revised version better?

Reply:
Oh, I see, Utf8Encoding.GetString eventually ends up in the same place.

Reply:
(I wonder if that means there's an opportunity to further improve Utf8Encoding.GetString to avoid multiple virtual dispatches that are likely resulting from inside the delegated call to CreateStringFromEncoding.)

Reply:
>  Utf8Encoding.GetString to avoid multiple virtual dispatches that are likely resulting from inside the delegated call to CreateStringFromEncoding

We would have to duplicate the code. I doubt that it is worth it.